.travis.yml

```diff
@@ -6,12 +6,13 @@ sudo: required
 dist: trusty
 osx_image: xcode8.3
 
-os: 
- - linux
- - osx
-
 matrix:
   include:
+    - os: osx
+
+# Default config (on trusty gcc4)
+    - os: linux
+
     - os: linux
```

Comment:
Moved to make it explicit. When we accept CircleCI builds we should drop at least some builds here to lower the load on travis.